### PR TITLE
fix AMS access documentation

### PR DESCRIPTION
**Pull request by ru-fu**
Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

File: manage-ams-access.md [hunk 1]
```diff
@@ -1,103 +1,56 @@
...
+### Self-signed certificates
 
-```bash
-$ openssl genrsa -out ca.key 2048
-$ openssl req -new -x509 -key ca.key -out ca.crt -days 1024
-```
+To use a self-signed certificate, complete the following steps:
 
-and tell AMS to trust it and verify all subsequents certificates.
+1. Invoke `amc` once on the client machine to generate the certificate.
```
Comment (adglkh) [inline manage-ams-access.md:21]: Simply invoking `amc` command won't create a key pair on the client machine. People need to explicitly run 
```
amc remote ls
```

Comment (ru-fu) [reply manage-ams-access.md]: Thanks, updated!

File: ams.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+The Anbox Management Service (AMS) sits at the heart of Anbox Cloud and handles all aspects of the application and container life cycle. It is responsible for managing containers, applications, addons, updates and more, ensuring high density, performance and fast container startup times.
+
+AMS is usually managed through the command line interface (the Anbox Management Client - AMC), which can run either on the same machine as AMS or on a remote machine.
+
+Since AMS exposes an HTTP interface, any tool can use the [AMS REST API](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801) to interact with AMS. Both the AMC (when running remotely) and the [Anbox Application Registry (AAR)](https://discourse.ubuntu.com/t/application-registry/17761) use this mechanism. You can also develop your own client.
```
Comment (morphis) [inline ams.md:5]: We maybe should link https://github.com/anbox-cloud/ams-sdk to give users some guidance on how to implement this.

**Review by morphis — APPROVED** [review]
LGTM, just one suggestion.

**Comment by adglkh** [comment]
LGTM

**Pull request merged**